x/liquidstakeibc/keeper/abci.go

```diff
@@ -18,17 +18,14 @@ import (
 func (k *Keeper) BeginBlock(ctx sdk.Context) {
 	// perform BeginBlocker tasks for each chain
 	for _, hc := range k.GetAllHostChains(ctx) {
-		if !hc.Active {
-			// don't do anything on inactive chains
-			continue
+		if hc.Active {
```

Comment:
Either add the other side of the IF statement or return, no ?

Reply:
want to keep processing the claims ( of already unstaking requests ) if any... and for that might require to keep channels active, etc.. so except staking all things are allowed for inactive chains..

Reply:
Understood